src/map/lua/lua_baseentity.cpp

```diff
@@ -17172,7 +17172,8 @@ void CLuaBaseEntity::drawIn(sol::variadic_args va)
         return;
     }
 
-    auto mobObj = dynamic_cast<CMobEntity*>(m_PBaseEntity);
+    auto   mobObj      = dynamic_cast<CMobEntity*>(m_PBaseEntity);
+    uint8  drawInRange = mobObj->GetMeleeRange() * 2;
```

Comment:
Where does this come from?

Reply:
Got it from the existing code for the draw in range, in battleutils.cpp on line 5626